include/wx/aui/auibar.h

```diff
@@ -560,6 +566,9 @@ class WXDLLIMPEXP_AUI wxAuiToolBar : public wxControl
     void SetMargins(int x, int y) { SetMargins(x, x, y, y); }
     void SetMargins(int left, int right, int top, int bottom);
 
+    void SetToolClientData (int tool_id, wxObject *client_data);
+    wxObject* GetToolClientData(int tool_id) const;
+
```

Comment:
Do we really need those? It looks like it would be enough to use `FindTool()` and similar (`ByPosition/Index` variants) and then call `{Get,Set}ClientData()` on the returned pointer, just as it's already apparently supposed to be done with `{Get,Set}UserData()`.

Reply:
I added them because wxToolBar has them, to keep coherency.